users/users/users.php

```diff
@@ -37,14 +41,158 @@ class UsersApiResourceUsers extends ApiResource
 	 */
 	public function delete()
 	{
-		$this->plugin->setResponse( JText::_( 'PLG_API_USERS_IN_DELETE_FUNCTION_MESSAGE' ));
+		$app              = JFactory::getApplication();
+		$db 			  = JFactory::getDbo();
+		$data             = array();
+		$eobj             = new stdClass;
+		$eobj->status     = false;
+		$eobj->id         = 0;
+
+		// Get values paased by put in the url
+		$data = $app->input->getArray(array());
+
+		// Check username or user_id to edit the details of user
+		if (isset($data['username']) || isset($data['user_id']))
+		{
+			if (!$data['user_id'] && isset($data['username']))
```

Comment:
Done change... Please confirm and revert with your feedback.

On Wed, Aug 31, 2016 at 1:38 PM, Ashwin Date <notifications@github.com>
wrote:

> In users/users/users.php
> <https://github.com/techjoomla/com_api-plugins/pull/36#discussion_r76941911>
> :
>
> > @@ -37,14 +41,158 @@ class UsersApiResourceUsers extends ApiResource
> >  	 */
> >  	public function delete()
> >  	{
> > -		$this->plugin->setResponse( JText::_( 'PLG_API_USERS_IN_DELETE_FUNCTION_MESSAGE' ));
> > +		$app              = JFactory::getApplication();
> > +		$db 			  = JFactory::getDbo();
> > +		$data             = array();
> > +		$eobj             = new stdClass;
> > +		$eobj->status     = false;
> > +		$eobj->id         = 0;
> > +
> > +		// Get values paased by put in the url
> > +		$data = $app->input->getArray(array());
> > +
> > +		// Check username or user_id to edit the details of user
> > +		if (isset($data['username']) || isset($data['user_id']))
> > +		{
> > +			if (!$data['user_id'] && isset($data['username']))
>
> Change this to just
>
> if (!$data['user_id'] && $data['username'])
>
> isset not needed since you have already put that check on the top
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/techjoomla/com_api-plugins/pull/36/files/72a0c7984ad7f2d66e03d260da1c62023b8b8e06#r76941911>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUNb4li2J992jVopiTaoOOd5dSLo2Ktvks5qlTZ4gaJpZM4JqyuW>
> .
>



-- 
Best Regards,
Deepali A. Madane